test/known_issues/test-symlinked-peer-modules.js

```diff
@@ -0,0 +1,78 @@
+/* eslint no-irregular-whitespace: 0 */
+'use strict';
+// Refs: https://github.com/nodejs/node/pull/5950
+
+// This test illustrates the problem that symlinked modules are unable
+// to find their peer dependencies. This was fixed in #5950 but that is
+// reverted because that particular way of fixing it causes too much
+// breakage (breakage that was not caught by either CI or CITGM on multiple
+// runs.
+
+const common = require('../common');
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+const exec = require('child_process').exec;
+
+if (common.isWindows) {
+  // On Windows, creating symlinks requires admin privileges.
+  // We'll only try to run symlink test if we have enough privileges.
+  exec('whoami /priv', function(err, o) {
+    if (err || o.indexOf('SeCreateSymbolicLinkPrivilege') == -1) {
+      console.log('Skipped: insufficient privileges');
+      return;
+    } else {
+      test();
+    }
+  });
```

Comment:
Because I don't call the symlinkSync until after other bits have been done and I'd rather skip it. I've refactored this tho, since the other work is all happening in a tmpdir and is pretty cheap it's no big deal.